src/hooks/use-modal-handler.ts

```diff
@@ -0,0 +1,12 @@
+import { useState } from "react";
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
IMHO u should use default export and there will be no need to use eslint-disable-next-line

Reply:
or we can just remove this lint setting

Reply:
done